FLIR/conservator/cli/cvc.py

```diff
@@ -45,7 +52,19 @@ def main(ctx, log, path):
         "CRITICAL": logging.CRITICAL,
     }
     logging.basicConfig(level=levels[log])
-    ctx.obj = path
+    if config:
+        try:
+            conservator = Conservator(Config.from_name(config))
+        except AttributeError:
+            raise RuntimeError(
+                f"Unknown/Invalid Conservator config '{config}'"
+            ) from None
+    else:
+        conservator = Conservator.default()
```

Comment:
Seeing as this code fragment is duplicated, you might consider making it a staticmethod

Alternatively: modify `Config.from_name` to handle `name="default"` with `Config.default()`, or raise an exception if not found. Then can change default name from None to "default" and replace these blocks with `Conservator(Config.from_name(config))`

Reply:
(but fine to merge as-is)

Reply:
The problem I had with unifying it back to one call is that default() has the other behavior of looking in the environment and the final fallback of prompting the user. It seems not so nice to hide that special behavior under the hood when `name="default"`. What do you think?

Reply:
Actually I was thinking about a new static method earlier, but didn't come up with a good name and skipped it. It occurs to me now that `Conservator.create()` would probably be suitable.

Reply:
I figured that as long as the special case of "default" was documented it would be ok. I guess we would want to call it something other than `from_name`. 

Anyways, let merge as-is and maaybe change if we are ever touching this/Config code again